pkg/external-boot-image/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+ARG KERNEL=
+ARG XENTOOLS=
+
+#FROM lfedge/eve-xen-tools:e36677ad441c2d5cd88cdd9831959eaa4715ab82 as initrd-build
+#FROM docker.io/lfedge/eve-kernel:eve-kernel-amd64-v6.1.38-generic-d2da815271a2-gcc as  kernel-build
+FROM $XENTOOLS as initrd-build
```

Comment:
@zedi-pramodh , we never used this approach before because we want to always fix the container version in order to keep the reproducibility of the build system. However, now that the kernel comes from outside our main tree, we knew this would happen sooner or later. Please, take a look on these links, where this approach was also discussed:

- https://github.com/moby/moby/issues/18119
- https://github.com/moby/moby/pull/31352

I think their approach is better and it can address the Yetus hadoling issue. The changes are minimal, you just need to use something like this (parsing the variables accordingly):

```Dockerfile
ARG KERNEL=latest
ARG XENTOOLS=latest

FROM lfedge/eve-xen-tools:$XENTOOLS as initrd-build
FROM lfedge/eve-kernel:$KERNEL as kernel-build
```